portfolio/src/App.tsx

```diff
@@ -1,23 +1,25 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import "./App.css";
+
+const profile: string = `I am a software developer with a strong passion for Quality Assurance and aspiration to grow my career as a software developer. 
```

Comment:
you dont actually need the `:string` because it will infer it automatically. Although, no issue with having it.